src/main/java/com/mixpanel/android/viewcrawler/EditProtocol.java

```diff
@@ -342,7 +343,13 @@ private Object convertArgument(Object jsonArgument, String type, List<String> as
             } else if ("float".equals(type) || "java.lang.Float".equals(type)) {
                 return ((Number) jsonArgument).floatValue();
             } else if ("android.graphics.drawable.Drawable".equals(type)) {
```

Comment:
can you do
`else if ("android.graphics.drawable.Drawable".equals(type) || "android.graphics.drawable.BitmapDrawable".equals(type)) {...}`